app/src/main/java/com/gholem/moneylab/features/add/adapter/AddTransactionsAdapter.kt

```diff
@@ -96,14 +98,28 @@ class AddTransactionsAdapter(
         }
 
     private fun mapToTransaction(
-        category: TransactionCategory?,
+        category: TransactionCategoryModel?,
         item: AddTransactionItem.Transaction
-    ) = Transaction(
+    ) = TransactionModel(
         category = category ?: listOfCategory.get(0),
-        amount = item.amount.toInt(),
+        amount = getAmount(item),
         date = item.date
     )
 
+    private fun getAmount(item: AddTransactionItem.Transaction): Int {
+        try{
```

Comment:
Do you need this try catch? Does the amount can be too big to be an `Int` value?